pkg/arvo/app/dojo.hoon

```diff
@@ -1134,8 +1134,14 @@
   ++  he-errd                                           ::  reject update
     |=  [rev=(unit sole-edit) err=@u]  ^+  +>
     =+  red=(fall rev [%nop ~])       ::  required for error location sync
+    =?  red  =(%mor -:red)
+      =/  rel  ;;((list sole-edit) +:red)
+      =/  idx  (find `(list sole-edit)`[%del `@ud`err]~ rel)
+      ?.  ?=(^ idx)
+        red
+      [%mor (oust [+:idx (lent rel)] rel)]
     =^  lic  say  (~(transmit sole say) red)
-    (he-diff %mor [%det lic] [%err err] ~)
+    (he-diff %mor [%det lic] [%err (lent buf.say)] ~)
```

Comment:
Since we cut out all edits/additions beyond the `err`th character in the buffer, isn't this equivalent to the original code?

The original code also has the flexibility to jump to error locations in the middle of the buffer, which may occur if you typed there and then hit return. (Unsure whether that actually hits this same codepath..)